tests/test_common.py

```diff
@@ -862,7 +862,7 @@ def test_tuner(
             "trainer_config__batch_size": randint(32, 64),
             "optimizer_config__optimizer": ["RAdam", "AdamW"],
         }
-    result = tuner.tune(
+    result, best_model = tuner.tune(
```

Comment:
If we include best model in the namedtuple, we dont need to change this as well.

Reply:
Changed.